files/en-us/web/api/rtcdatachannelevent/index.html

```diff
@@ -3,21 +3,34 @@
 slug: Web/API/RTCDataChannelEvent
 tags:
   - API
-  - Experimental
   - Interface
   - RTCDataChannelEvent
   - Reference
   - WebRTC
   - datachannel
 browser-compat: api.RTCDataChannelEvent
 ---
-<p>{{APIRef("WebRTC")}}{{SeeCompatTable}}</p>
+<p>{{APIRef("WebRTC")}}</p>
 
-<p>The <strong><code>RTCDataChannelEvent()</code></strong> constructor returns a new {{domxref("RTCDataChannelEvent")}} object, which represents a {{domxref("datachannel")}} event. These events sent to an {{domxref("RTCPeerConnection")}} when its remote peer is asking to open an {{domxref("RTCDataChannel")}} between the two peers.</p>
+The <strong><code>RTCDataChannelEvent</code></strong> is an interface
+representing an event related to a specific {{DOMxRef("RTCDataChannel")}}.
```

Comment:
maybe better wording:

```suggestion
The <strong><code>RTCDataChannelEvent</code></strong> interface
represents an event related to a specific {{DOMxRef("RTCDataChannel")}}.
```

?